deploy.go

```diff
@@ -194,6 +196,28 @@ func RunDeploy(c *cli.Context) error {
 	return nil
 }
 
+func displayNewCommits(owner string, repo string, c *cli.Context, client *github.Client) {
+	ref := Ref(c.String("ref"), git.Head)
+
+	opt := &github.DeploymentsListOptions{
+		Environment: c.String("env"),
+	}
+
+	deployments, _, error := client.Repositories.ListDeployments(owner, repo, opt)
+	if error == nil {
+		sha := *deployments[0].SHA
```

Comment:
Should we handle the case where `len(deployments) == 0`? This will panic right now if we deploy to a new environment or deploy a new repo.